src/ja/server/database/sql/database.py

```diff
@@ -196,6 +196,7 @@ def update_job(self, job: Job) -> None:
                                                                     0, 0),
                                          machine=None)
             session.add(job_entry)
+            # job.uid = 
```

Comment:
How could I get the id of the newly added job entry before adding it? Since the id has to be assigned to the job before adding it to the DB

Reply:
the id is assigned automatically, uid is not, you can't get the id before adding.